src/Parser/Rogue/Subtlety/Modules/Features/SymbolsOfDeathUptime.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import { formatPercentage } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Enemies from 'Parser/Core/Modules/Enemies';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+
+class SymbolsOfDeathUptime extends Analyzer {
+  static dependencies = {
+    enemies: Enemies,
+    combatants: Combatants,
+  };
+
+  statistic() {
+    const symbolsOfDeathUptime = this.combatants.selected.getBuffUptime(SPELLS.SYMBOLS_OF_DEATH.id) / this.owner.fightDuration;
+    return (
+      <StatisticBox
+        icon={<SpellIcon id={SPELLS.SYMBOLS_OF_DEATH.id} />}
+        value={`${formatPercentage(symbolsOfDeathUptime)} %`}
+        label={(
+          <dfn data-tip={'Symbols of Death uptime'}>
+          Symbols of Death uptime
```

Comment:
Only include tooltips if they add new information (which this one does not). Also, 'uptime' is one word.